Firestore/Example/SwiftBuildTest/main.swift

```diff
@@ -288,9 +288,11 @@ func types() {
     let _: Firestore;
     let _: FirestoreSettings;
     let _: GeoPoint;
+    let _: ListenerRegistration;
```

Comment:
Can you delete the comment above about how this list was generated, since it's no longer true?